scripts/check-cocoapods-locks.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Early exit when script is run on platforms other than macOS
+if [[ "$OSTYPE" != "darwin"* ]]; then
+  exit 0
+fi
+
+green="tput setaf 2"
+yellow="tput setaf 3"
+blue="tput setaf 4"
+reset="tput sgr0"
+
+HEADER_PRINTED=false
```

Comment:
```suggestion
headerPrinted=false
```
(all caps for env variables?)